### ISSUE TITLE
Scene with 3 cameras shows duplicate objects in UI

### ISSUE THREAD

**Issue by reed-alpert** [googlevr/gvr-instant-preview#11]
My scene has 3 cameras - 1 for game objects, and 2 for the UI. Objects are assigned to cameras via Layers. When I use IP and bring up the UI (via the AppButton) I see one UI in the player (as usual) but 2 UIs (and 2 lasers) on the phone (see attached photos). 

The IP config can be seen in the player image.

![player](https://user-images.githubusercontent.com/23744336/31732071-2ea23e1c-b405-11e7-8cfe-99dda5e82a6f.jpg)
![phone](https://user-images.githubusercontent.com/23744336/31732076-3148e562-b405-11e7-8873-97c77e9626d3.jpg)

**Comment by gstanlo** [comment]
This might be related to an issue in which extra "eye" cameras are added to the scene if the head cameras get enabled/disabled. Do you see anything odd with your camera arrangement in your scene?

**Comment by gstanlo** [comment]
If it looks like this is the case, we have a potential fix which might work for you. I can send it to you if you email me: stanlo@google.com

**Comment by reed-alpert** [comment]
Yes, the cameras increase in number when the UI is opened (see attached images). The 2 UI cameras are disabled most of the time, and only enabled when the UI is open.

With UI
![with_ui](https://user-images.githubusercontent.com/23744336/31752010-b64dbcf0-b456-11e7-984a-14c6f165ae59.jpg)

With-out UI
![without_ui](https://user-images.githubusercontent.com/23744336/31752012-b8bba3da-b456-11e7-8efd-12d224c54b50.jpg)

**Comment by fredsa** [comment]
This should be fixed in an upcoming release of the Google VR SDK for Unity.
